components/Marker/Marker.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "./Marker.module.scss";
+
+const Marker = (props: any) => {
+  const { color, name, id } = props;
```

Comment:
It looks like this library requires us to return text in the map marker. We'll need to extract the `text` prop and use it in the return statement like:

```tsx
<div
  className="marker"
  style={{ backgroundColor: color, cursor: "pointer" }}
  title={name}
>{text}</div>
```

Reply:
Go this in!